### ISSUE TITLE
Getting `Unexpected token '}'` 

### ISSUE THREAD

**Issue by efimk-lu** [serverless/serverless#7554]
# Bug Report

## Description
Getting `Unexpected token '}'`
1. What did you do?
sls deploy
1. What happened?
Got the following error
```
/usr/lib/node_modules/serverless/node_modules/serverless-tencent-tools/sdk/debug/lib/wshub-client/index.js:87
  }
  ^

SyntaxError: Unexpected token '}'
    at wrapSafe (internal/modules/cjs/loader.js:1072:16)
    at Module._compile (internal/modules/cjs/loader.js:1122:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/usr/lib/node_modules/serverless/node_modules/serverless-tencent-tools/sdk/debug/lib/client.js:1:30)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
```

I'm not using `serverless-tencent-tools` in my project
1. What should've happened?
Successful build
1. What's the content of your `serverless.yml` file?
```
service:
  name: dash-resources

custom:
  env: ${opt:env, env:USER}
  prefix: ${self:custom.env}_${self:service.name}

provider:
  name: aws
  stage: ${self:custom.env}
  region: ${opt:region, 'us-west-2'}
  stackName: ${self:custom.env}-${self:service.name}
  environment:
    ENV: ${self:custom.env}
  stackTags:
    stack-name: ${self:provider.stackName}
    region: ${self:provider.region}
    env: ${self:custom.env}

resources:
  Description: "Dashboard resources"
  Resources:
    DashboardBucket:
      Type: AWS::S3::Bucket
      Properties:
        BucketName: my-name
        AccessControl: PublicRead
        WebsiteConfiguration:
          IndexDocument: index.html
          ErrorDocument: index.html
    DashboardS3BucketPolicy:
          Type: AWS::S3::BucketPolicy
          Properties:
            Bucket:
              Ref: DashboardBucket
            PolicyDocument:
              Statement:
                - Sid: PublicReadGetObject
                  Effect: Allow
                  Principal: "*"
                  Action:
                  - s3:GetObject
                  Resource:
                    Fn::Join: [
                      "", [
                        "arn:aws:s3:::",
                        {
                          "Ref": "DashboardBucket"
                        },
                        "/*"
                      ]
                    ]
```
1. What's the output you get when you use the `SLS_DEBUG=*` environment variable (e.g. `SLS_DEBUG=* serverless deploy`)

Similar or dependent issues:
N/A

**Comment by dallinwright** [comment]
Same here, just trying to login or create a new item and it errors out. Not serverless yaml, nothing.

**Comment by efimk-lu** [comment]
And may I ask what is `serverless-tencent-tools`?

**Comment by Jorer** [comment]
Duplicate of #7555

**Comment by jarvisluong** [comment]
As from a quick work around, I needed to manually fix the syntax error from the file which caused the issue, I made a diff here: https://gist.github.com/jarvisluong/2e94d2c8882482fd56816affae90c190

the file to be patched is: /usr/local/lib/node_modules/serverless/node_modules/serverless-tencent-tools/sdk/debug/lib/wshub-client/index.js

**Comment by tobilg** [comment]
Downgrading to `1.64.1` temporarily fixes the problem, see https://github.com/serverless/serverless/issues/7555#issuecomment-610861821

**Comment by tobilg** [comment]
A bugfix was just merged in https://github.com/serverless/serverless/pull/7556, now waiting for the new version to be released

**Comment by Declow** [comment]
@efimk-lu 
From what I can understand serverless-tencent-tools is an SDK for Tencent Cloud. Tencent is a Chinese cloud provider.
https://serverless.com/framework/docs/providers/tencent/

**Comment by tobilg** [comment]
Serverless v1.67.3 was released: https://www.npmjs.com/package/serverless/v/1.67.3 Works for me...

**Comment by Declow** [comment]
I have no problem with building using Serverless v1.67.3

**Comment by IrmantasM** [comment]
This broke all my builds today, can I somehow remove serverless-tencent-tools?

**Comment by medikoo** [comment]
Closing as duplicate of https://github.com/serverless/serverless/issues/7555

Was fixed with v1.67.3 (which is no longer immune to eventual errors in `serverless-tencent-tools`)
